Formula/k/kafkactl.rb

```diff
@@ -18,9 +18,10 @@ class Kafkactl < Formula
   depends_on "go" => :build
 
   def install
+    ENV["CGO_ENABLED"] = "0"
```

Comment:
`CGO_ENABLED` is for cross-compilation, while homebrew go builds on the target architecture 

```suggestion
```

Reply:
`CGO_ENABLED` allows to link C libraries to go code.
Locally I had some issues when installing the formula from source with cgo enabled.
Since the application is written in pure go, there is no need to enable it. With CGO disabled, it builds faster and it also aligns with how the application is compiled outside the homebrew ecosystem. So I would rather keep it like it is.

Reply:
> it also aligns with how the application is compiled outside the homebrew ecosystem

Unless it causes some issue, we dont really care how it builds outside homebrew ecosystem